proposals/apple-mdm/Fleet-MDM-Plan.md

```diff
@@ -0,0 +1,218 @@
+# Fleet MDM Plan
```

Comment:
@chris-mcgillicuddy https://fleetdm.com/handbook/digital-experience#how-to-write-headings-subheadings

> Fleet MDM plan

etc

Reply:
@mikermcneil Wasn't sure whether we considered these proper nouns. I'll update accordingly.

Reply:
:+1: We should keep a list of terms of art that we treat as proper nouns (e.g. "Fleet Desktop", "Fleet Sandbox").  These are the extreme exception though: when in doubt, assume not unless it's a true proper noun, such as "Ricky Martin" or "Narnia".